src/lib/aqua.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright 2022 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import assert from "node:assert";
+
+import {
+  AquaConfig,
+  CompilationResult,
+  Aqua,
+  Call,
+  Input,
+  Path,
+} from "@fluencelabs/aqua-api/aqua-api";
+import type { callFunctionImpl } from "@fluencelabs/fluence/dist/internal/compilerSupport/v3impl/callFunction";
+
+export type Data = Parameters<typeof callFunctionImpl>[4];
```

Comment:
We can make a type and export it in FJS for the `data` argument

Reply:
This can be done. Maybe together with changing `callFunctionImpl` to accept an object with some of the fields that are optional, instead of 5 arguments each of which is required